MapboxDirections/MBRouteOptions.swift

```diff
@@ -119,9 +155,21 @@ open class RouteOptions: DirectionsOptions {
             URLQueryItem(name: "alternatives", value: String(includesAlternativeRoutes)),
             URLQueryItem(name: "continue_straight", value: String(!allowsUTurnAtWaypoint))
         ])
-
-        if includesExitRoundaboutManeuver {
-            queryItems.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
+        
+        switch profileIdentifier {
+        case .automobile, .automobileAvoidingTraffic:
+            if includesExitRoundaboutManeuver {
+                queryItems.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
+            }
+            
+        case .walking:
+            queryItems.append(URLQueryItem(name: "alley_bias", value: String(alleyBias)))
+            queryItems.append(URLQueryItem(name: "walkway_bias", value: String(walkwayBias)))
+            queryItems.append(URLQueryItem(name: "walking_speed", value: String(walkingSpeed)))
+        case .cycling:
+            break
+        default:
+            break
         }
 
         if !roadClassesToAvoid.isEmpty {
```

Comment:
`"exclude"` (roadClassesToAvoid) should only be appended if profile identifier is any of `[.automobile, .automobileAvoidingTraffic, .cycling]` for now.